segmentation_processing/mask_rcnn/utils.py

```diff
@@ -0,0 +1,128 @@
+from typing import Any, Tuple
+
+import numpy as np
+import torch
+
+INSTANCE_CATEGORIES = [
```

Comment:
Is there a reason why this is defined here and in `instance_names.py`?

Reply:
The Mask RCNN's instances names could differ from the STEGO ones. All I did was relocate model code within its given directory I didn't try to unify between the two.